docs/source/topics/concept.rst

```diff
@@ -0,0 +1,136 @@
+Simulation Concept
+==================
+
+Background
+----------
+
+.. TODO::
+
+    HEP context.
+
+Components
+----------
+
+The core simulation builds on several components, and concepts:
+
+* :term:`Job Generator`,
+* :term:`Job Queue`,
+* :term:`Pools <Pool>` and their :term:`Controllers <Controller>`,
+* :term:`Drones <Drone>`, and
+* the :term:`Scheduler`,
+
+If you are planning to adapt the simulation for your specific use case, please
+consider the different components to determine what and where to extend functionality.
+
+Job Generator
+~~~~~~~~~~~~~
+
+The Job Generator works on the job input files. It takes care to
+translate time-based characteristics of the :term:`jobs <Job>` into simulation
+time. For this the timestamp of the first :term:`job` of each job input file is
+taken as the ``base`` timestamp, resulting in a time value of ``0`` for the
+first :term:`job`. All following :term:`jobs <Job>` are adapted accordingly,
+i.e. time is ``time - base``.
+
+The Job Generator itself acts as a generator, meaning that a :term:`job` is put
+into the simulations :term:`Job Queue` as soon as the simulation time corresponds
+to the translated :term:`job` queueing time.
+
+Job Queue
+~~~~~~~~~
+
+The Job Queue is filled with :term:`jobs <Job>` in creation-time order by the
+:term:`Job Generator`. The queue is managed by the :term:`scheduler` and contains
+all :term:`jobs <Job>` that are not yet scheduled to a :term:`drone` as well as
+:term:`jobs <Job>` that have not yet been processed succesfully.
+
+Pools
+~~~~~
+
+Pools are created based on the pool input files. Each pool is characterised by
+a set of defined resources. Further, pools have a ``capacity`` number of
+:term:`drones <Drone>` that can be created from a given pool. If the capacity
+is not specified, a maximum capacity of ``float("inf")`` is assumed.
+
+For pools, we differentiate static and dynamic pools. While static pools are
+intialised with a fixed amount of :term:`drones <Drone>`, the number of
+:term:`drones <Drone>` is adapted dynamically by the
+:term:`pool controller <Controller>` for dynamic pools.
+
+.. autoclass:: lapis.pool.Pool
+.. autoclass:: lapis.pool.StaticPool
+
+Controllers
+~~~~~~~~~~~
+
+Each :term:`pool` is started with an appropriate controller. Each controller runs
```

Comment:
```suggestion
Each :term:`pool` is managed by a parent controller. Each controller runs
```

Reply:
I wouldn't like to put the information about parent here... This is a piece of information that we do not cover. What do you think about:

```suggestion
Each :term:`pool` is managed by a controller. Each controller runs
```